src/css/custom.css

```diff
@@ -127,7 +127,7 @@
 }
 
 .text-slate\/60 {
```

Comment:
I think the rule name is a little odd: it feels like this was the name of a specific color. Should we consider renaming this rule if it's going to apply a theme color instead?